src/record/record-core.ts

```diff
@@ -117,13 +117,6 @@ export class RecordCore<Context = null> extends Emitter {
   })
 }
 
-  /**
-  * Removes scope from emitter
-  */
-  public removeContext (ref: any): void {
```

Comment:
this is references in other places, is it meant to be deleted?

Reply:
I added that method on the latest realease, it wasn't present on previous versions. Since I did a rollback it got removed again. I think when it is called in other places the prototype chain calls the removeContext method from Emitter, since Record Core is an extension of it. We might keep it though on record core for clarity?